lab3.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int search (int a[],int n) {
+    
+    int b=0;
+    for (int i=0; i<n; i++) {
+        b=b^a[i];    
+    }
+    return b;
+}
+
+int main() {
+    int a[]= {1,2,3,2,1,};
+    
+    int n = sizeof(a)/sizeof(int);
```

Comment:
sizeof(int) - what do you mean here? and replace n to once_num
```suggestion
    int once_num = sizeof(input_arr)/sizeof(int);
```